py3status/modules/weather_owm.py

```diff
@@ -0,0 +1,797 @@
+    icon_rain: Icon for rain
+        (default '🌧')
+    icon_snow: Icon for snow
+        (default '❄')
+    icon_sun: Icon for sunshine
+        (default '☼')
+    icon_temp: Icon for temperature
+        (default '○')
+    icon_thunderstorm: Icon for thunderstorms
+        (default '⛈')
+    icons: A dictionary relating weather code to icon.
+        See https://openweathermap.org/weather-conditions for a complete list
+        of supported icons. This will fall-back to the listed icon if there is
+        no specific icon present.
+        There are multiple ways to specify individual icons based on the id:
+            - Use the key 'i601' to reference the condition with id = 601
+              (snow)
+            - Use the key 'i230_i232' to reference a span of conditions
+              inclusive, in this case conditions (230, 231, 232) (thunderstorm
+              with drizzle)
+        Also, you can specify the icons for the various formatting sections
+        below. For example, to specify the icon for 'format_pressure', use
+        'pressure'. A few formatting sections will take from the defaults or be
+        dynamic, including
+            - clouds
+            - rain
+            - wind
+            - sunrise
+            - sunset
+        These may be specified regardless.
+        (default None)
+    lang: An ISO 639-1 code for your language (two letters)
+        (default 'en')
+    request_timeout: The timeout in seconds for contacting the IP API.
+        (default 10)
+    temperature_color: Color the temperature output based on a color scale.
+        For reference, see https://goo.gl/NGbjIE
+        (default False)
+
+Format Placeholders:
+ - All:
+   - {icon}: The icon associated with a formatting section
+ - format_cloud:
+   - {coverage}: Cloud coverage percentage.
+ - format_humidity:
+   - {humid}: The humidity percentage
+ - format_pressure:
+   - {press}: The measurement of current atmospheric pressure in Pascals
+   - {sea_level}: The measurement of current atmospheric pressure at
+     sea-level in Pascals.
+ - format_rain:
+   - {cm}: Measurement in centimeters.
+   - {ft}: Measurement in feet.
+   - {in}: Measurement in inches.
+   - {mm}: Measurement in millimeters.
+   - {m}: Measurement in meters.
+   - {yrd}: Measurement in yards.
+ - format_snow:
+   - {cm}: Measurement in centimeters.
+   - {ft}: Measurement in feet.
+   - {in}: Measurement in inches.
+   - {mm}: Measurement in millimeters.
+   - {m}: Measurement in meters.
+   - {yrd}: Measurement in yards.
+ - format_temp:
+   - {c_max}: The maximum Celsius temperature.
+   - {c_min}: The minimum Celsius temperature.
+   - {c}: Current Celsius temperature reading.
+   - {f_max}: The maximum Fahrenheit temperature.
+   - {f_min}: The minimum Fahrenheit temperature.
+   - {f}: Current Fahrenheit temperature reading.
+   - {k_max}: The maximum Kelvin temperature.
+   - {k_min}: The minimum Kelvin temperature.
+   - {k}: Current Kelvin temperature reading.
```

Comment:
personally I'd do theses as `{temp}` `{temp_min}` etc and then allow the user to select a temp scale eg `F`, `C` or `K`

Also same with wind speed etc